sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -651,7 +651,7 @@
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: ["e2e_bootup_bad_rom_ext_signature"]
```

Comment:
this should be empty, otherwise it shows up in the DV dashboard.

Reply:
Done